dice.js

```diff
@@ -0,0 +1,16 @@
+
+const rollDice = function(){
+    const randNums = function (){
+        let n1=1, n2=1;
+
+        n1 += Math.round(5*Math.random());
+        n2 += Math.round(5*Math.random());
+
+        return [n1, n2, n1+n2]
+    }
```

Comment:
Great solution; Also there another way to create a random number withen a range:
```Math.round(Math.random() * (max - min) + min);```